geodocker-base/Dockerfile.template

```diff
@@ -10,11 +10,11 @@ ARG ACCUMULO_VERSION
 ARG THRIFT_VERSION
 
 # JAVA
```

Comment:
I submit a PR with some code from my fork for this template, should help with updating the java version in future updates